src/catalog/es-AR.json

```
@@ -1191,7 +1095,9 @@
       "download-reports": "Download de Informes",
       "sku-supply-report": "Reporte de stock de SKUs",
       "let-me-know-report": "Reporte de avisame",
-      "integration-errors-report": "Errores de integración VTEX x KPL"
+      "integration-errors-report": "Errores de integración VTEX x KPL",
+      "request-sent-with-warning": "¡Solicitud enviada! <br /> Cuando el proceso haya finalizado, se enviará un correo electrónico a '+ email +' con un enlace para descargar la hoja. <br /> Recuerde que puede llevar demasiado tiempo si el total de registros es grande.",
```

the pattern for this string `' + email + '` is different from the other languages. Are you parsing it after in the code to insert the email? This one in es-AR seems to be lacking spaces.
```suggestion
      "request-sent-with-warning": "¡Solicitud enviada! <br /> Cuando el proceso haya finalizado, se enviará un correo electrónico a ' + email + ' con un enlace para descargar la hoja. <br /> Recuerde que puede llevar demasiado tiempo si el total de registros es grande.",
```